src/components/editor/NavigationModal.tsx

```diff
@@ -6,6 +8,34 @@ import { IconButton } from "~/components/ui/IconButton";
 
 type Props = Modal.Props;
 
+const Td: React.FC<{ className?: string; rowSpan?: number }> = ({ children, className, rowSpan }) => {
+  return (
+    <td className={classNames("p-3 pb-1", className)} rowSpan={rowSpan}>
+      {children}
+    </td>
+  );
+};
+
+const Th: React.FC<{ className?: string }> = ({ children, className }) => {
+  return <td className={classNames("font-bold", className)}>{children}</td>;
+};
+
+const SmallText: React.FC = ({ children }) => {
+  return <span className="text-xs text-gray-400 leading-snug inline-block">{children}</span>;
+};
+
+const InteractionImage: React.FC<{ alt: string; src: string }> = ({ alt, src }) => {
+  return (
+    <td className="align-bottom">
+      <div className="pb-6 w-full">
+        <div className="h-16 w-full relative">
+          <Image alt={alt} layout="fill" objectFit="contain" src={src} />
```

Comment:
This breaks on my computer. Requires width/height and doesn't know fill/objectFit.

Pretty sure `<Image>` is useless for svgs.